osrparse/dump.py

```diff
@@ -47,44 +47,53 @@ def dump_timestamp(replay):
 
 def dump_replay_data(replay):
     replay_data = ""
-    for event in replay.play_data:
+    for event in replay.replay_data:
+        t = event.time_delta
         if isinstance(event, ReplayEventOsu):
-            replay_data += f"{event.time_delta}|{event.x}|{event.y}|{event.keys.value},"
+            replay_data += f"{t}|{event.x}|{event.y}|{event.keys.value},"
         elif isinstance(event, ReplayEventTaiko):
-            replay_data += f"{event.time_delta}|{event.x}|0|{event.keys.value},"
+            replay_data += f"{t}|{event.x}|0|{event.keys.value},"
         elif isinstance(event, ReplayEventCatch):
-            replay_data += f"{event.time_delta}|{event.x}|0|{int(event.dashing)},"
+            replay_data += f"{t}|{event.x}|0|{int(event.dashing)},"
         elif isinstance(event, ReplayEventMania):
-            replay_data += f"{event.time_delta}|{event.keys}|0|0,"
-
-    filters = [{"id": lzma.FILTER_LZMA1, "dict_size": 1 << 21, "mode": lzma.MODE_FAST}]
-    compressed = lzma.compress(replay_data.encode("ascii"), format=lzma.FORMAT_ALONE, filters=filters)
+            replay_data += f"{t}|{event.keys.value}|0|0,"
+
+    filters = [
```

Comment:
the `dict_size` and `mode` should be open for a consumer, with parameters in the dumping methods in `Replay`, and through this function